src/StripeTests/Properties.cs

```diff
@@ -0,0 +1 @@
+[assembly: Xunit.CollectionBehavior(DisableTestParallelization = true)]
```

Comment:
There has to be a way to not disable this and have it work right?

Reply:
Like clarifying: not specifically in this PR per se, it's fine to merge as is, but surprising there isn't a feature for this in the lib

Reply:
Sure, but I suspect the way may be "use a lock to ensure thread-safety", which would negate most gains from parallelization since tests would need to wait on the lock to send requests. I might revisit this in the future but for now I didn't want to block on this since the suite still runs very fast.

Reply:
Actually, I think I know what's happening: stripe-dotnet uses a single `HttpClient` object (which is what Microsoft recommends -- the `SendAsync` method is thread-safe). However, if multiple tests are run in parallel, then we cannot assert on the last request since this may happen:
- Thread 1: reset mock
- Thread 1: send request
- Thread 2: reset mock
- Thread 1: assert on last request -> fails because the mock was reset

Even if we don't reset the mock between every test, this may happen:
- Thread 1: send request
- Thread 2: send request
- Thread 1: assert on last request -> fails because the last request is not the expected one

So I think disabling parallel execution is the right move here.